gcd/src/main/java/org/spine3/server/storage/datastore/DsProperties.java

```diff
@@ -162,7 +162,7 @@ static void makeEventFieldProperties(Event event,
         builder.set(producer_id.toString(), producerId);
     }
 
-    static Predicate<Entity> activeEntityPredicate() {
+    static Predicate<Entity> activedEntityPredicate() {
```

Comment:
Something is wrong with English in this name.

And also, why is it in the `DsProperties`? Does not seem to be a good place for such a common thing.